Automattic/jetpack
# Hide Core's default My Sites menu on WP.com

related to https://github.com/Automattic/jetpack/pull/37993, https://github.com/Automattic/jetpack/pull/36632

## Proposed changes:


Since we changed not to load some CSS in https://github.com/Automattic/jetpack/pull/37993, the Core's "My Sites" menu [was shown](https://github.com/Automattic/jetpack/pull/37993#issuecomment-2185445456) on Simple Classic sites. We want to fix this to prevent users from feeling that the site is a multisite. 

This PR changed to hide the "My Sites" menu on Simple Classic sites.

| before | after |
|--------|--------|
| <img width="360" alt="Screenshot 2024-06-24 at 17 08 18" src="https://github.com/Automattic/jetpack/assets/5287479/2f6e5987-66a6-42f8-aeb5-bf77477602e9"> | <img width="259" alt="Screenshot 2024-06-24 at 17 07 32" src="https://github.com/Automattic/jetpack/assets/5287479/fa900e90-0da2-484d-9f88-b88605c39d50"> | 

This PR also fixes the bug where "My Sites" was visible on mobile. 

| before | after |
|--------|--------|
| <img width="217" alt="Screenshot 2024-06-24 at 17 08 08" src="https://github.com/Automattic/jetpack/assets/5287479/f68a3a51-6f3b-4029-88a7-56009857ef0a"> | <img width="193" alt="Screenshot 2024-06-24 at 17 07 42" src="https://github.com/Automattic/jetpack/assets/5287479/d97ad0e2-46ad-48ca-9a79-344a3c89164f"> | 

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Apply this PR to a Simple Classic site using your sandbox https://github.com/Automattic/jetpack/pull/38000#issuecomment-2185854469
* Use `wpsh` to add the option: `update_option("is_fully_managed_agency_site", 1)`
* Ensure the "My Sites" menu is hidden
